hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/AssignmentManager.java

```diff
@@ -2028,4 +2036,41 @@ private void addToPendingAssignment(final HashMap<RegionInfo, RegionStateNode> r
   MasterServices getMaster() {
     return master;
   }
+
+  public Map<byte[], Pair<ServerName, Set<ServerName>>> getProblematicRegions() {
```

Comment:
this method needs a javadoc description of what we're returning

Reply:
Added the javadoc in new patch.